### DISCUSSION TITLE
Piggy bank on multiple accounts

### DISCUSSION THREAD

**Discussion by maxsmooth** [firefly-iii/firefly-iii#9284] [Firefly III] [answered]
Hello everybody.
I have a 2000 € piggy bank on two account, 1000 € on a bank account and 1000 € on a saving account.
How can i manage it on firefly (firefly wants only one account)?
(Sorry for my bad english)

**Comment by JC5** [comment, marked as answer]
Great question! This is not yet possible, but it is on my list to improve.

**Reply by maxsmooth** [reply]
Great news! It's the only thing that i miss in firefly.

**Reply by maxsmooth** [reply]
A "Use all asset accounts" entry would also be fine
